plugins/woocommerce/src/Internal/Utilities/DatabaseUtil.php

```diff
@@ -157,10 +160,12 @@ public function format_object_value_for_db( $value, string $type ) {
 				$value = strval( $value );
 				break;
 			case 'date':
```

Comment:
I wondered if it might be better to keep this clause (for the `'date'` type) as it is, and instead add a new clause for new type `'date_gmt'` (then update the relevant column mappings). 

Since this only seems to be used in relation to COT/HPOS, though, and since all the relevant date columns in our COT tables are GMT, I decided changing the way `'date'` works would be cleaner.

Reply:
I think `date_gmt` is better. Explicit is always better than implicit and since WP has both local and GMT dates, it can easily lead to confusion. That said, it's a minor problem in the context of this PR.